src/signing/networkMessages/kusama.js

```diff
@@ -46,11 +46,11 @@ export async function ClaimRewardsTx(senderAddress) {
   let allClaimingTxs = []
   const api = await getAPI()
   const stakerRewards = await api.derive.staking.stakerRewards(senderAddress)
-
-  if (stakerRewards.length === 0) {
+  const newStakerRewards = stakerRewards.filter(({ era }) => era.words[0] > 718)
```

Comment:
```suggestion
  const newStakerRewards = stakerRewards.toJSON().filter(({ era }) => era > 718)
```

tried this?

Reply:
It doesn't work...

<img width="440px" src="https://user-images.githubusercontent.com/40721795/81445934-2a4dc980-917a-11ea-82fa-3e1290079a79.png">

Reply:
true cause it is an array

Reply:
```suggestion
  const newStakerRewards = stakerRewards.filter((reward) => reward.toJSON().era > 718)
```